packages/migrations/src/actions/2025.01.07T00.00.00.breaking-changes-request-count.ts

```diff
@@ -0,0 +1,19 @@
+import { type MigrationExecutor } from '../pg-migrator';
+
+export default {
+  name: '2025.01.07T00.00.00.breaking-changes-request-count.ts',
+  run: ({ sql }) => sql`
+CREATE TYPE
+  breaking_change_formula AS ENUM('PERCENTAGE', 'REQUEST_COUNT');
+
+ALTER TABLE
+  targets
+ADD COLUMN
+  validation_request_count INT NOT NULL DEFAULT 1;
```

Comment:
I just wanted to comment that we should not add a default value but, but I double checked 😄 
> From PostgreSQL 11, adding a column with a constant default value no longer means that each row of the table needs to be updated when the ALTER TABLE statement is executed. Instead, the default value will be returned the next time the row is accessed, and applied when the table is rewritten, making the ALTER TABLE very fast even on large tables.